dusty/systems/docker/testing_image.py

```diff
@@ -0,0 +1,43 @@
+from . import get_docker_client
+
+
+def _ensure_testing_spec_base_image(testing_spec):
+    if 'image' in testing_spec and 'build' in testing_spec:
+        raise RuntimeError('Only 1 of `image` and `build` keys are allowed in testing spec')
```

Comment:
Def need that specs class to consolidate stuff like this, you really don't want validation spread all throughout your app